src/server/goog-device/mw/WebsocketProxyOverAdb.ts

```diff
@@ -340,6 +358,7 @@ export class WebsocketProxyOverAdb extends WebsocketProxy {
             })
             .catch((e) => {
                 this.logger.error(e);
+                Utils.captureMessage('Failed to run setUpTest', 'Android', this.udid);
```

Comment:
@metocks // iOS쪽과 동일하게 수정해 보았습니다.
```suggestion
            const self = this;
            Sentry.captureException(e, scope => {
                 scope.setTags('ramiel_device_type', 'Android');
                 scope.setTags('ramiel_device_id', self.udid);
                 scope.setTags('ramiel_message', e.ramiel_message);
                 return scope;
            });
```